### ISSUE TITLE
You must create an endpoint before performing this operation

### ISSUE THREAD

**Issue by sbozzie** [sdimkov/hubot-skype-web#13]
Skype-web is not logging on, with the error message "You must create an endpoint before performing this operation" repeated in the hubot log.

**Comment by sbozzie** [comment]
A reinstall of hubot now leads to skype-web login errors:

```
C:\hubot>bin\hubot
[Wed Dec 16 2015 16:50:31 GMT+0000 (GMT Standard Time)] ERROR SkypeWeb adapter failed to login!
[Wed Dec 16 2015 16:50:32 GMT+0000 (GMT Standard Time)] ERROR Error: SkypeWeb adapter failure in initial login!  at Object.error (C:\hubot\node_modules\hubot-skype-web\src\skypeweb.coffee:56:17) at [object Object]._onTimeout (C:\hubot\node_modules\hubot-skype-web\src\skypeweb.coffee:115:20)  at Timer.listOnTimeout (timers.js:92:15)
```

I've checked passwords, and I can log-in fine as this user though the web interface.

**Comment by TheAznShumai** [comment]
I can verify I'm also getting the same error.

**Comment by sbozzie** [comment]
A second bot using this plugin had crashed (for other reasons) and now cannot login: I'm assuming that Skype have changed something in the login process?

**Comment by sdimkov** [comment]
Hi guys, I'll try to help on this tomorrow

**Comment by velichkin** [comment]
I have the same error. I've tried to debug something and as I can see, login process was really changed by skype.

**Comment by sbozzie** [comment]
Patch at https://github.com/nagodon/hubot-skype-web/commit/db4d68f6351047ee14a91ffddf6c1c790778b51b resolved this for me.

**Comment by TheAznShumai** [comment]
The Patch at nagodon@db4d68f helped get into the hubot shell which I assume means I'm logged into skype.  I'm wondering, is anyone else having problem receiving messages from Skype?  I'm kind of new at this and I can't seem to get this to work.  What I noticed is that even though, I get the shell to run without any issues, I still can't see my hubot skype-id go online.  Any help would be appreciated.

**Comment by sdimkov** [comment]
@TheAznShumai your skype-id will likely not go online. This adapter does not update online presense. You should test this by directly writing to your skype hubot and look for feedback

**Comment by sdimkov** [comment]
fix applied in 0.9.11
